input-output-hk/daedalus
# [DDW-1061] Downloaded installer being left behind after wallet is updated to latest version

This PR intends to fix the problem where we have downloaded installers being left behind after Daedalus is updated.

## Screenshots

No UI changes.

## Local Testing Steps:

1. Change `package.json` to `4.8.0`.
2. Run the local environment.
3. Wait for the download to finish. You don't have to go through the installation as it will just install the Testnet without the updated logic.
4. Make sure you have the finished installer in the `Daedalus Testnet` state directory downloads folder.
5. Stop the environment.
6. Change `package.json` to `4.9.0`.
7. Run the local environment.
8. Open the downloads folder in state directory. The installer shouldn't be there anymore.

## Testing Checklist



- [Slack QA thread](https://input-output-rnd.slack.com/messages/GGKFXSKC6)

---

## Review Checklist

### Basics

- [ ] PR assigned to the PR author(s)
- [ ] `input-output-hk/daedalus-dev` and `input-output-hk/daedalus-qa` assigned as PR reviewers
- [ ] If there are UI changes, Alexander Rukin assigned as an additional reviewer
- [ ] All visual regression testing has been reviewed (assign `run Chromatic` label to PR to trigger the run)
- [ ] PR has appropriate labels (`release-vNext`, `feature`/`bug`/`chore`, `WIP`)
- [ ] PR link is added to a Jira ticket, ticket moved to In Review
- [ ] PR has a good description that summarizes all changes
- [ ] PR contains screenshots (in case of UI changes)
- [ ] CHANGELOG entry has been added to the top of the appropriate section (_Features_, _Fixes_, _Chores_) and is linked to the correct PR on GitHub

### Code Quality

- [ ] Code is properly typed with typescript types

### Testing


### After Review
